test/storage/offline_database.test.cpp

```diff
@@ -34,69 +35,60 @@ void deleteFile(const char* name) {
     }
 }
 
-void writeFile(const char* name, const std::string& data) {
-    mbgl::util::write_file(name, data);
-}
-
 void copyFile(const char* orig, const char* dest) {
-    mbgl::util::write_file(dest, mbgl::util::read_file(orig));
+    util::write_file(dest, util::read_file(orig));
 }
 
 } // namespace
 
-TEST(OfflineDatabase, TEST_REQUIRES_WRITE(Create)) {
-    using namespace mbgl;
+class OfflineDatabaseTest : public ::testing::Test {
```

Comment:
I dislike the "fixture" style of testing -- it makes it non-obvious what tests do, because they have hidden behavior without even an obvious function call you can trace through. Can we stick to the existing style?